cmd/tailscale/cli/debug.go

```diff
@@ -190,6 +190,11 @@ var debugCmd = &ffcli.Command{
 			Exec:      runDebugDERP,
 			ShortHelp: "test a DERP configuration",
 		},
+		{
+			Name:      "capture",
+			Exec:      runCapture,
+			ShortHelp: "streams pcaps to stdout for debugging",
```

Comment:
"to stdout" ?  make that a flag, with "-" meaning stdout?  if you do a filename, you need explicit "-" support for macOS which can't write to disk when sandboxed.

Reply:
Can't someone just pipe it to a file if they want a stored capture? That way we don't have to deal with invariants like MacOS.